tests/python_tests/test_continuous_batching.py

```diff
@@ -82,13 +74,14 @@ def test_e2e_real_models(tmp_path, model_id):
 @pytest.mark.parametrize("prompt", batched_prompts[1:])  # num_beams=15 diverges on the first prompt.
 @pytest.mark.precommit
 def test_continuous_batching_vs_stateful(prompt, generation_config):
-    model_id, path, tokenizer, model, stateful = read_model((
-        "facebook/opt-125m",
-        Path("opt-125m")
-    ), padding_side="left")
-    cb_pipe = get_continuous_batching(path)
+    model_id = "facebook/opt-125m"
+    _, _, models_path = download_and_convert_model(model_id, padding_side="left")
+    ov_pipe = create_ov_pipeline(models_path)
+    cb_pipe = create_ov_pipeline(models_path, pipeline_type=PipelineType.PAGED_ATTENTION)
```

Comment:
```suggestion
    cb_pipe = create_ov_pipeline(models_path, pipeline_type=PipelineType.CONTINIOUS_BATCHING)
```
as tests in current file are designated for `ContinuousBatchingPipeline`